sympy/physics/mechanics/system.py

```diff
@@ -1,8 +1,1077 @@
-from sympy.core.backend import eye, zeros, Matrix
-from sympy.physics.vector import dynamicsymbols
-from sympy.physics.mechanics.functions import find_dynamicsymbols
+from functools import wraps
 
-__all__ = ['SymbolicSystem']
+from sympy.core.backend import Basic, ImmutableMatrix, Matrix, eye, zeros
+from sympy.core.containers import OrderedSet
+from sympy.physics.mechanics.actuator import ActuatorBase
+from sympy.physics.mechanics.body_base import BodyBase
+from sympy.physics.mechanics.functions import (
+    Lagrangian, _validate_coordinates, find_dynamicsymbols)
+from sympy.physics.mechanics.joint import Joint
+from sympy.physics.mechanics.kane import KanesMethod
+from sympy.physics.mechanics.lagrange import LagrangesMethod
+from sympy.physics.mechanics.loads import _parse_load, gravity
+from sympy.physics.mechanics.method import _Methods
+from sympy.physics.mechanics.particle import Particle
+from sympy.physics.vector import Point, ReferenceFrame, dynamicsymbols
+from sympy.utilities.iterables import iterable
+
+__all__ = ['SymbolicSystem', 'System']
+
+
+def _reset_eom_method(method):
+    """Decorator to reset the eom_method if a property is changed."""
+
+    @wraps(method)
+    def wrapper(self, *args, **kwargs):
+        self._eom_method = None
+        return method(self, *args, **kwargs)
+
+    return wrapper
+
+
+class System(_Methods):
```

Comment:
This "system" seems specific to the joints concepts. I'm thinking "JointsSystem" may be a better name and allows us to reserve the the more general name "System" for a more general system concept.

Reply:
I see in one of the tests that you can pass in bodies that have been manually oriented without joints. So disregard.